k8s.gcr.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -131,6 +133,7 @@
     "sha256:b01168405a440797a2e0002eb72a91f116f03a2fdc26a2088ab1f87788de0d92": ["v0.3.19"]
     "sha256:06e82e7708d7ab5a3923deb0b72bc29f67892e2870075c4a785b6b7bebd77ded": ["v0.4.0-beta.0"]
     "sha256:b56992e76106efef56e271981e2fcd196282424b23f4c0b8adec3405cd7ba0f7": ["v0.4.0-beta.1"]
+    "sha256:99ef6dfd8db1d1ad4ed324c216d27720a4bdd7c1d07b0fe02970977b8fdb55e0": ["v0.4.0-rc.0"]
```

Comment:
wrong SHA here @vincepri 

should be `sha256:411cbcf381a725797b4a3745d255934f9478e4a551e71bbaab88b85913d46bb6`

Reply:
Fixed, thanks!